dkron/store.go

```diff
@@ -627,7 +627,16 @@ func (s *Store) Shutdown() error {
 	return s.db.Close()
 }
 
-func (s *Store) unmarshalExecutions(items []*kv, stopWord string) ([]*Execution, error) {
+func (s *Store) unmarshalExecutions(items []*kv, jobName string) ([]*Execution, error) {
+	// Get the job's timezone
+	var jobLocation *time.Location
+	if job, err := s.GetJob(jobName, nil); err == nil {
```

Comment:
Could we avoid going to the store while unmarshaling?

Reply:
I'm not too thrilled about that either. Avoiding it might require a lot of plumbing - but I'll see what can be done about it.

Reply:
How about this:
Commit 1d614c3 refactors things so the timezone gets passed down to unmarshalExecutions.
Now, only the dashboard goes to the store to retrieve the timezone.